_account-security/security/encryption/ssh-for-amazon-web-services.md

```diff
@@ -0,0 +1,231 @@
+
+          You can create a **new** Security Group or update an existing one. For this tutorial, we'll create a new group.
+
+          1. In the **Assign a security group** field, select **Create a new security group**.
+          2. In the **Security group name** field, enter a unique name for the Security Group. For example: `Stitch Bastion`
+          3. In the table below the **Description** field, you'll add Stitch's IP addresses to the security group:
+             - **Type**: If the default SSH port for your server is 22, set this to **SSH**.
+
+                If it’s something else, set this to **Custom TCP Rule**.
+             - **Protocol**: This will default to **TCP** - leave it as-is.
+             - **Port Range**: This is the number of the SSH port associated with the server. **If you selected SSH as the Type**, this will default to `22`.
+
+                **If you selected Custom TCP Rule**, enter the number of the SSH port in this field.
+             - **Source**: This should default to **Custom**. In the field next to the Source drop-down menu, paste one of the following IP addresses:
+
+                {% for ip-address in ip-addresses %}
+                - {{ ip-address.ip }}
+                {% endfor %}
+
+          4. Click the **Add Rule** button to add another rule.
+          5. Repeat steps 3 and 4 until all of Stitch's IP addresses have been added to the Security Group.
+
+             Here's what the Security Group rules should look like:
+
+             ![Configuring the EC2 Instance Security Group]({{ site.baseurl }}/images/shared/ssh/amazon-ec2-security-group.png)
+          6. When finished, click the **Review and Launch** button in the lower right corner of the page.
+
+      - title: "Review and launch the SSH server"
+        anchor: "review-launch-ssh-server"
+        content: |
+          The last step is to review the settings for the SSH server and launch it.
+
+          Review the instance's settings, paying particular attention to the fields highlighted in the image below:
+
+          ![Reviewing the EC2 Instance Details]({{ site.baseurl }}/images/shared/ssh/amazon-ec2-instance-review.png)
+
+          - The **Security Groups** section should list either a new Security Group for Stitch OR an existing group that contains group rules for Stitch's IP addresses. If it doesn't, [refer to Step 2.2](#ec2-instance-security-group) for instructions.
+          - In the **Instance Details** section:
+             - **Network**: This field should contain the ID of the same VPC that the database is in.
+             - **Subnet**: This field should contain the ID of a public subnet.
+             - **Assign Public IP & Assign IPv6 IP:** We strongly recommend using a [public subnet with the instance](#configure-ec2-instance) and auto-assigning a public IP address. This will ensure that Stitch can access the instance.
```

Comment:
this isn't just a strong recommendation, I think it's a requirement

Reply:
Agreed. Thanks for pointing it out; it slipped by me.